spec/controllers/api/application_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe API::ApplicationController, type: :controller do
+  controller do
+    def index
+      render json: {}, status: :ok
```

Comment:
Any reason we're stubbing the controller actions in these tests?

If the serializers were changed or broken, we'd want the suite to fail for sure, and I don't think that's the case right now.

Reply:
This controller is acting as an anonymous controller in the tests which essentially just means that it extends `API::ApplicationController`. I had to use the anonymous controller here because we don't actually have any actions in `API::ApplicationController` and I wanted to test the before actions of application controller.